pkg/distributor/distributor.go

```diff
@@ -45,6 +45,21 @@ var (
 		Name:      "distributor_received_samples_total",
 		Help:      "The total number of received samples.",
 	}, []string{"user"})
+	incommingSamples = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
s/incomming/incoming

Reply:
What do you think of updating the help text in `receivedSamples` to say that it only includes samples that have not been deduped?

Reply:
Honestly I think this metric should be `receivedSamples` and the current `receivedSamples` should be `acceptedSamples` or `storedSamples`.

Reply:
:+1: Good idea, that naming pattern is much more clear. I like `acceptedSamples` if the metric is increased before the push, and `storedSamples` if it is only increased when there are no errors writing to the ingesters. Which of those two options do you think would be more valuable?